graphql/e2e/schema/introspection.graphql

```diff
@@ -0,0 +1,113 @@
+query {
```

Comment:
Could we have this file at one place and reference it from everywhere else? I thought we already had it somewhere else too.